server/pulp/server/db/fields.py

```diff
@@ -48,3 +50,16 @@ def to_python(self, value):
         """
         ret = super(UTCDateTimeField, self).to_python(value)
         return dateutils.ensure_tz(ret)
+
+    def _validate(self, value, **kwargs):
+        """
+        Ensures that the value is a datetime.datetime object that has tzinfo set.
+
+        :param value: datetime.datetime object
+        :param kwargs: any extra kwargs that should be passed to the super method
+        :raises: mongoengine.errors.ValidationError
```

Comment:
Consider adding a newline separating the inputs and outputs of the function's docstring.